src/cursor/cur_backup.c

```diff
@@ -204,6 +205,14 @@ __curbackup_close(WT_CURSOR *cursor)
         __wt_verbose(
           session, WT_VERB_BACKUP, "%s", "Releasing resources from forced stop incremental");
         __wt_backup_destroy(session);
+        /*
+         * We need to force a checkpoint to the metadata to make the force stop durable. Without it,
+         * the backup information could reappear if we crash and restart.
+         */
```

Comment:
Should this behavior appear in the doc?  That forcing incremental to stop has a side effect of a checkpoint?

Reply:
It is not a full checkpoint. It is only writing the turtle file. It is effectively the same thing as reconfiguring compatibility which also writes the turtle file.